build.ps1

```diff
@@ -122,6 +122,7 @@ $projects = @(
     "y2j",
     "wmi-adapter",
     "resources/brew",
+    "reboot_pending",
```

Comment:
this needs to be moved to `$windows_projects` along with wmi-adapter

Reply:
Updated.